service/application/src/test/java/com/solace/maas/ep/event/management/agent/realTests/NegativeTerraformEVMRConnectedTests.java

```diff
@@ -0,0 +1,572 @@
+        validateErrorCommandResponse(commandResponse);
+    }
+
+    @Test
+    public void sempVersionTooOldTest() {
+        testOutputMdTitle2(getTestMethodName());
+
+        String correlationId = "myCorrelationId3";
+        String context = "abc123";
+        String oldServiceId = "v0r806w0bmj";
+        CommandMessage commandResponse = runTest("addQueue.tf", oldServiceId, correlationId, context);
+        validateErrorCommandResponse(commandResponse);
+    }
+
+    @Test
+    public void badProviderTest() {
+        testOutputMdTitle2(getTestMethodName());
+
+        String correlationId = "myCorrelationId3";
+        String context = "abc123";
+        CommandMessage commandResponse = runTest("badProvider.tf", mainServiceId, correlationId, context);
+        validateErrorCommandResponse(commandResponse);
+    }
+
+    @Test
+    public void badMessagingServiceIdTest() {
+        testOutputMdTitle2(getTestMethodName());
+
+        String correlationId = "myCorrelationId1";
+        String context = "abc123";
+        CommandMessage commandResponse = runTest("addQueue.tf", "imABadServiceId", correlationId, context);
+        validateErrorCommandResponse(commandResponse);
+    }
+
+    @Test
+    public void badCommandTypeInCommand() {
+        testOutputMdTitle2(getTestMethodName());
+
+        String correlationId = "myCorrelationId3";
+        String context = "abc123";
+        CommandMessage commandResponse = runTest("addQueue.tf", mainServiceId, correlationId, context, this::createCommandMessageBadCommandType);
+        validateErrorCommandResponse(commandResponse);
+    }
+
+
+    @Test
+    public void missingParametersInCommand() {
+        testOutputMdTitle2(getTestMethodName());
+
+        String correlationId = "myCorrelationId3";
+        String context = "abc123";
+        CommandMessage commandResponse = runTest("addQueue.tf", mainServiceId, correlationId, context, this::createCommandMessageMissingParameters);
+        validateErrorCommandResponse(commandResponse);
+    }
+
+    @Test
+    public void hclCommandIsNotBase64Encoded() {
+        testOutputMdTitle2(getTestMethodName());
+
+        String correlationId = "myCorrelationId3";
+        String context = "abc123";
+        CommandMessage commandResponse = runTest("addQueue.tf", mainServiceId, correlationId, context, this::createCommandMessageHclNotBase64Encoded);
+        validateErrorCommandResponse(commandResponse);
+    }
+
+    @Test
+    public void hclHasVariableThatIsNotSetInTerraformPlugin() {
+        testOutputMdTitle2(getTestMethodName());
+
+        String correlationId = "myCorrelationId1";
+        String context = "abc123";
+        CommandMessage commandResponse = runTest("addQueueNewVariableNotInPlugin.tf", mainServiceId, correlationId, context);
+        validateErrorCommandResponse(commandResponse);
+    }
+
+    @Test
+    public void manyRequestsForTheSameContextAtTheSameTime() {
+        String context = "abc123";
```

Comment:
Looks like context is same value in all tests, can we move it to the `@BeforeEach` section?